src/ai/backend/common/web/session/__init__.py

```diff
@@ -11,7 +11,8 @@
 
 from ai.backend.common.logging import BraceStyleAdapter
 
-log = BraceStyleAdapter(logging.getLogger("ai.backend.web.server"))
+logger = logging.getLogger("ai.backend.web.server")
```

Comment:
@hydroxyde Is there a reason that you divide one-liner code to two liners? `logger` looks like it never used after the instantiation and one reference.